performance_manager/startup.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python
+
+import argparse
+import json
+import logging
+import os
+import pathlib
+import sched
+import sys
+import time
+
+from typing import Dict, List
+
+import sqlalchemy as sa
+from sqlalchemy.orm import sessionmaker
+
+from lib import (
+    SqlBase,
+    MetadataLog,
+    get_experimental_engine,
+    get_local_engine,
+    process_vehicle_positions,
+)
+
+logging.getLogger().setLevel("INFO")
+
+DESCRIPTION = """Entry Point to RDS Manipulating Performance Manager Scripts"""
+HERE = os.path.dirname(os.path.abspath(__file__))
+
+
+def load_environment() -> None:
+    """
+    boostrap .env file for local development
+    """
+    try:
+        if int(os.environ.get("BOOTSTRAPPED", 0)) == 1:
+            return
+
+        env_file = os.path.join(HERE, "..", ".env")
+        logging.info("bootstrapping with env file %s", env_file)
+
+        with open(env_file, "r", encoding="utf8") as reader:
+            for line in reader.readlines():
+                line = line.rstrip("\n")
+                line.replace('"', "")
+                if line.startswith("#") or line == "":
+                    continue
+                key, value = line.split("=")
+                logging.info("setting %s to %s", key, value)
+                os.environ[key] = value
+
+    except Exception as exception:
+        logging.error("error while trying to bootstrap")
+        logging.exception(exception)
+        raise exception
+
+
+def parse_args(args: list[str]) -> argparse.Namespace:
+    """parse args for running this entrypoint script"""
+    parser = argparse.ArgumentParser(description=DESCRIPTION)
+    parser.add_argument(
+        "--experimental",
+        action="store_true",
+        dest="experimental",
+        help="if set, use a sqllite engine for quicker development",
+    )
+
+    parser.add_argument(
+        "--interval",
+        default=60,
+        dest="interval",
+        help="interval to run event loop on",
+    )
+
+    parser.add_argument(
+        "--seed",
+        action="store_true",
+        dest="seed",
+        help="if set, seed metadataLog table with paths",
+    )
+
+    return parser.parse_args(args)
+
+
+def seed_metadata_table(sql_session: sessionmaker) -> None:
```

Comment:
seed metadata function. it would be cool if it took a filepath.

might be a good idea to have this in a different script so we can insert stuff into the db on the fly while startup dot py is running.